File: packages/nx-micronaut-maven/src/generators/application/files/java/src/main/java/__packageDirectory__/HelloController.java
package <%= packageName %>;

import io.micronaut.http.MediaType;
import io.micronaut.http.annotation.Controller;
import io.micronaut.http.annotation.Get;
import io.micronaut.http.annotation.Produces;

@Controller("/hello") // <1>
public class HelloController {
    @Get // <2>
    @Produces(MediaType.TEXT_PLAIN) // <3>
    public String index() {
        return "Hello World"; // <4>
    }
}

File: packages/nx-micronaut-maven/src/generators/application/files/kotlin/src/main/kotlin/__packageDirectory__/HelloController.kt
package <%= packageName %>

import io.micronaut.http.MediaType
import io.micronaut.http.annotation.Controller
import io.micronaut.http.annotation.Get
import io.micronaut.http.annotation.Produces

@Controller("/hello") 
class HelloController {

    @Get
    @Produces(MediaType.TEXT_PLAIN) 
    fun index() = "Hello World"
}